### PR TITLE
feat: header menu selections

### PR DESCRIPTION
**Pull request by cbt1**
Skärmavbild 2023-09-08 kl  12 32 07

### CODE REVIEW

File: src/pivot-table/components/cells/DimensionTitleCell.tsx [hunk 1]
```diff
@@ -75,10 +85,18 @@ const DimensionTitleCell = ({
     [cell, isDim],
   );
 
-  const sortFromMenu = async (evt: React.MouseEvent, newSortDirection: SortDirection) => {
-    evt.stopPropagation();
-    await changeSortOrder(headerData, newSortDirection);
-  };
+  const sortRelatedArgs = useMemo(() => {
```
Comment (cbt1) [inline src/pivot-table/components/cells/DimensionTitleCell.tsx:88]: Might be a bit overkill to wrap these props in `useMemo`. I wrapped the `HeadCellMenu` component in the util repo with `React.memo`. Think I need some more testing to ensure that it actually reduces the number of commited renders.

Comment (a-m-dev) [reply src/pivot-table/components/cells/DimensionTitleCell.tsx:88]: +1

File: src/pivot-table/components/cells/DimensionTitleCell.tsx [hunk 2]
```diff
@@ -45,6 +47,13 @@ const labelTextStyle: React.CSSProperties = {
 
 export const testId = "title-cell";
 
+const handleHeadCellMenuKeyDown = () => {};
```
Comment (a-m-dev) [inline src/pivot-table/components/cells/DimensionTitleCell.tsx:50]: This is optional now with use of (v2.0.1). you can skip it

**Review by a-m-dev — APPROVED** [review]
🚀

File: src/pivot-table/components/cells/DimensionTitleCell.tsx (hunk 1 above)
Comment (cbt1) [reply src/pivot-table/components/cells/DimensionTitleCell.tsx]: It did not change the number of renders. I've now removed useMemo.

**Pull request merged**